tests/common_testing/alfasim_sdk_common_testing/case_builders.py

```diff
@@ -147,3 +147,21 @@ def build_compressor_pressure_table_description(
         pressure_ratio_table=pressure_ratio_table,
         isentropic_efficiency_table=isentropic_efficiency_table,
     )
+
+
+def build_constant_pvt_table(
+    energy_model_primary_variable: constants.EnergyModelPrimaryVariable,
+):
+    if (
+        energy_model_primary_variable
+        == constants.EnergyModelPrimaryVariable.Temperature
+    ):
+        return case_description.PvtModelTableParametersDescription.create_constant()
+    elif energy_model_primary_variable == constants.EnergyModelPrimaryVariable.Enthalpy:
+        return (
+            case_description.PvtModelTableParametersDescription.create_constant_ph_table()
+        )
+    else:  # pragma no cover
+        assert isinstance(
+            energy_model_primary_variable, constants.EnergyModelPrimaryVariable
+        ), f"{energy_model_primary_variable} not supported."
```

Comment:
If we add a new value to `constants.EnergyModelPrimaryVariable` This will not trigger an error even if there is we forget to add code for handling it, a better approach will be just fail:
```suggestion
        assert False, f"{energy_model_primary_variable!r} not supported"
```